public/styles/coffeepage.css

```diff
@@ -0,0 +1,149 @@
+#coffeePageContainer{
```

Comment:
General CSS comments:

You're not very DRY. You've targeted many elements by their ID, and given too many specific rules to each element. You should rather think of this from CSS -> HTML rather than HTML -> CSS.

One way to accomplish a refactor would be to take all of the duplicate CSS rules, starting with the most common,  pull them out into their own `.class { prop: val; }` rule, and then apply those rules to the corresponding elements on the page. 

It's generally better practice to accomplish CSS through class-based rules rather than one element at a time through ID's.